requirements.txt

```diff
@@ -0,0 +1,53 @@
+addict==2.2.0
```

Comment:
i think the requirements should be included in the pipfile instead? cc @fantix

Reply:
Oh, yes please if possible. You may use `pipenv install xxx~=x.x` to add them into `Pipfile`. Please let me know if there're difficulties or issues, we may find an approach to work it out. :gratitude: